example/example.json

```diff
@@ -88,6 +88,14 @@
             "title": "Comments",
             "ui:widget": "textarea",
             "ui:height": 5     
+        },
+        "ibans": {
+            "type": "array",
+            "title": "Iban",
+            "items": {
+                "type": "string"
+            },
+            "ui:widget": "input"
```

Comment:
I think `inputs` is a better name for this widget, since it renders multiple input fields.